src/lib/Microsoft.Fx.Portability.Reports.DGML/ReferenceNode.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Fx.Portability.Reporting.ObjectModel;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace Microsoft.Fx.Portability.Reports.DGML
+{
+    class ReferenceNode
+    {
+        private bool _searchInGraph = false;
+
+        public List<TargetUsageInfo> UsageData { get; set; }
+
+        public string Assembly { get; set; }
```

Comment:
`public string Assembly { get; }`
and same with any other members below that are only set in the constructor